redfishtool/Systems.py

```diff
@@ -73,6 +74,7 @@ def displayOperations(self,rft):
         print("                               -- <targetVal> =None|Pxe|Floppy|Cd|Usb|Hdd|BiosSetup|Utilities|Diags|UefiTarget|")
         print("     Processors [list]         -- get the \"Processors\" collection, or list \"id\" and URI of members.")
         print("      Processors [IDOPTN]        --  get the  member specified by IDOPTN: -i<id>, -m<prop>:<val>, -l<link>, -a #all")
+        print("     Inventory [list]          -- get the \"Inventory\" collection, or list \"id\" and URI of members.")
```

Comment:
Should also update README.md similarly to show this option in the "Chassis Operations" section.